### PR TITLE
Add InfoListItem Examples

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes 4371.

#### Changes proposed in this Pull Request:

- Add InfoListItem Examples
-
-

#### Screenshots / Screen Recording (if applicable)
Screenshot 2023-10-06 at 11 21 04 AM

- 

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: components/kitchen-sink.tsx [hunk 1]
```diff
@@ -245,6 +247,48 @@ export const KitchenSink: React.FC = (): JSX.Element => {
                     />
                 </Card.Content>
             </Card>
+            <Card style={styles.card}>
+                <Card.Title title="InfoListItem" />
+                <Card.Content style={{ alignItems: 'center' }}>
+                    {/* With Info prop */}
+                    <InfoListItem
+                        title={'Title'}
+                        subtitle={'A subtitle'}
```
Comment (surajeaton) [inline components/kitchen-sink.tsx:256]: Only `Subtitle` looks better instead of `A subtitle`

Comment (surajeaton) [reply components/kitchen-sink.tsx:256]: For all the InfoListItem examples

**Review by surajeaton — COMMENTED** [review]
prettier is failing
Please run `yarn prettier`

**Comment by manojleaton** [comment]
Please resolve conflicts

**Comment by jvgreiner** [comment]
looks like yarn lock needs updated

**Review by ArshdeepSingh3-eaton — APPROVED** [review]

**Pull request merged**